theano/gpuarray/tests/test_opt.py

```diff
@@ -138,11 +138,21 @@ def test_local_gpualloc_memset_0():
     ones = numpy.ones((2,), dtype='float32')
 
     # Test with 0 from CPU op.
+    # Should not be transfered as the only client is the output
```

Comment:
Does this apply for all the node where the input has only one client and it is the output ?
This condition
node.inputs[0].clients[0] == 'output'